.eslintrc.js

```diff
@@ -31,7 +31,16 @@ module.exports = {
 		"no-negated-condition": "off",
 		"require-jsdoc": "off",
 		"no-mixed-requires": "off",
-		"max-len": ["warn", 120],
+		"max-len": [
+			"error",
+			{
+				"code": 120,
+				"comments": 200,
+				"ignoreTrailingComments": true,
```

Comment:
Does this mean that you can put a comment with any length after a line of code?
I don't know whether this makes sense. Or maybe I don't understand the setting.

Reply:
Removed